### PR TITLE
ps-5.6-5922

### PR DESCRIPTION
**Pull request by patrickbirch**
Updated utility-user document based on Zsolt's PR comments

### CODE REVIEW

File: doc/source/management/utility_user.rst [hunk 1]
```diff
@@ -44,11 +46,11 @@ Option :variable:`utility_user_password` specifies the password for the utility
 
 Option :variable:`utility_user_schema_access` specifies the name(s) of the schema(s) that the utility user will have access to read write and modify. If a particular schema named here does not exist on start up it will be ignored. If a schema by the name of any of those listed in this option is created after the server is started, the utility user will have full access to it.
 
- Example: :option:`--utility_user_schema_access` =schema1,schema2,schema3 ;
+ Example: :option:`--utility_user_schema_access` =schema1,schema2,schema3;
```
Comment (dutow) [inline doc/source/management/utility_user.rst]: The semicolon isn't needed for the parameter at all

File: doc/source/management/utility_user.rst [hunk 2]
```diff
@@ -44,11 +46,11 @@ Option :variable:`utility_user_password` specifies the password for the utility
...
 
 Option :variable:`utility_user_privileges` allows a comma-separated list of extra access privileges to grant to the utility user.
 
- Example: :option:`--utility-user-privileges` ="CREATE, DROP, LOCK TABLES"
+ Example: :option:`--utility-user-privileges` ="CREATE_DROP,LOCK TABLES"
```
Comment (dutow) [inline doc/source/management/utility_user.rst]: Typo: CREATE,DROP

File: doc/source/management/utility_user.rst [hunk 3]
```diff
@@ -10,7 +10,9 @@ This user has a mixed and special scope of abilities and protection:
 
   * Utility user will not appear in the mysql.user table and can not be modified by any other user, including root.
 
-  * Utility user will not appear in :table:`USER_STATISTICS`, :table:`CLIENT_STATISTICS` or :table:`THREAD_STATISTICS` tables.
+  * Utility user will increase the results in the `SHOW STATUS` for `Connections`, `Threads_connected` and `Threads_created`.
+  
+  * Utility user will not appear in :table:`USER_STATISTICS`, :table:`CLIENT_STATISTICS` or :table:`THREAD_STATISTICS` tables or in any performance_schema tables.
```
Comment (dutow) [inline doc/source/management/utility_user.rst]: `performance_schema` doesn't need a tag or something?

**Review by dutow — COMMENTED** [review]
LGTM other than those typos

File: doc/source/management/utility_user.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/management/utility_user.rst]: removed the underscore and there is no space between words.

File: doc/source/management/utility_user.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/management/utility_user.rst]: Added a link.

**Review by george-lorch — APPROVED** [review]

File: doc/source/management/utility_user.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/management/utility_user.rst]: removed.

**Pull request merged**